app/models/ping_thing_recent_stat.rb

```diff
@@ -27,7 +28,8 @@ class PingThingRecentStat < ApplicationRecord
     :min,
     :network,
     :num_of_records,
-    :p90
+    :p90,
+    :average_slot_latency
```

Comment:
If it does not affect anything, you could sort that fields in alphabetical order